src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+
     public String getRandomColor() {
-        return null;
+        int number = new Random().nextInt(Colors.values().length);
+        return Colors.values()[number].toString();
```

Comment:
Use `name()` for getting String representation of enum constants instead of `toString()`.